profiles/common/modules/features/nexteuropa_multilingual/nexteuropa_multilingual.install

```diff
@@ -61,15 +61,12 @@ function nexteuropa_multilingual_install() {
   variable_set('pathauto_node_pattern', 'content/[node:source:title]');
 
   // Remove old language switcher block.
+  // @TODO is not working
```

Comment:
After the platform installation I run 
```
$context = module_invoke('context', 'load', 'site_wide');
dpm([$context]);
```
In results I wasn't able to see 'language' in the $context->reactions['block']['blocks'] path.
Maybe more details on what is not working could help to understand better this problem.